cluster_client.cpp

```diff
@@ -234,14 +249,24 @@ void cluster_client::handle_cluster_slots(protocol_response *r) {
             if (strcmp(addr, m_connections[j]->get_address()) == 0 &&
```

Comment:
@YaacovHazan How often is this used (I did not follow all code paths)? I wonder if there's a point to optimize connection references with a map.  In general, we need to be wary of performance regressions...

Reply:
We not using it so often.
We call it once when memtier starts to get cluster topology, and then only when we got MOVED errors